src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -38,7 +38,7 @@ public class ImageView : View
             var imageView = (ImageView)bindable;
             string url = (string)newValue;
             url = (url == null ? "" : url);
-            if (imageView.IsCreateByXaml && url.Contains("*Resource*"))
+            if ((imageView.IsCreateByXaml || (imageView.GetParent()?.IsCreateByXaml ?? false)) && url.Contains("*Resource*"))
```

Comment:
Is there a case that the parent is not ceated by xaml but the chlid is from xaml?

Reply:
If crate button by xaml, icon of button is created code. so button.Icon.IsCreateByXaml is false.
Problem is that IconUrl is not drawing on xaml editor.
```csharp
<Button x:Name="btnDashBoard2" Position="0,75" Text="Dashboard" PointSize="8" TextColor="#007EF9" WidthResizePolicy="FillToParent" IconURL="*Resource*/profile.png" Size="184,26" Margin="0,0,10,10" Opacity="Infinity"/>
```

Reply:
If so checking only a parent won't solve all problems, especially if the component is made of 3 depth objects.

Reply:
As far as I know, switch's thumb is 3depth object.
Is there a way to set IsCreateByXaml to true for all sub-components?

Reply:
Right, I also think this issue. but there is no idea about that.
How about deleting check IsCreateByXaml?, I mean just only check the resource name if it is contain '*Resource*'.

Reply:
How about override `IsCreateByXaml` of Button to set sub-component's `IsCreateByXaml` to true? (Switch is also a Button)

Reply:
I've updated my patch. I've added to override 'IsCreateByXaml' in Control.

Reply:
Is it possible to add child to parent after parent's IsCreateByXaml is set to be true?
(the added child's IsCreateByXaml is false as @rabbitfor mentioned above)

If so, isn't it required to set child's IsCreateByXaml whenever child is added?

Reply:
IsCreateByXaml is set to true at the time it creates an object by xaml. If you add child after the parent's IsCreateByXaml is set, I think the added object has no problem with false.

Reply:
Hi, Mr. Eun,
I talked with Mr. Fang, we suggest another way to reach it, like the following snippets.

`
            if (url.StartWith("*Resource*"))
            {
                string resource = Tizen.Applications.Application.Current.DirectoryInfo.Resource;
                url = url.Replace("*Resource*", resource);
            }
`

In this way, user can use '*Resource*' way to set the URL of ImageView/BackgoundImage both in c# codes and XAML.
We don't need set child's IsCreateByXaml of NUI Components control when it create by xaml.

Reply:
@rabbitfor @Jaehyun-Cho could you check teng's opinion? I think it is good way to fix.

Reply:
I think @elishateng and @AdunFang 's idea is great :)

Reply:
@rabbitfor @Jaehyun-Cho @elishateng @AdunFang 
Please check my new patch.